include/secure_services.h

```diff
@@ -60,8 +60,8 @@ int spm_request_random_number(u8_t *output, size_t len, size_t *olen);
  * @param[in]  num         Number of bytes to copy.
  *
  * @retval 0        If successful.
- * @retval -EINVAL  If destination or destination is 0, or if num is <= 0.
- * @retval -EPERM   If source is not inside any of the allowed ranges.
+ * @retval -EINVAL  If destination is NULL, or if num is <= 0.
```

Comment:
I think these changes are in the wrong commit

Reply:
Fixed :)